### ISSUE TITLE
Save workspace when switching to another bookmark

### ISSUE THREAD

**Issue by thrucker** [szw/vim-ctrlspace#150]
I'm working with multiple projects (i.e. bookmarks) and would like to automatically save the current workspace when switching to another workspace of another bookmark.

Suppose I'm in workspace **main** of project **A**. The workspace contains files **x** and **y**. Now I open file **z** as well.

After finishing my work in file **z** I want to do something else in project **B**. So I switch via `<C- > b` to Project **B** and load workspace **other_workspace** via `w`.

When done in project **B** I switch back to workspace **main** of project **A** and would expect that files **x**, **y** and **z** are open, but **z** is actually missing.

**Comment by szw** [comment]
Yeah, indeed that would make sense. Workspace autosaving could work also before workspace loading.
